### PR TITLE
feat(service-portal):  add audio and video player

### PR DESCRIPTION
**Pull request by thorkellmani**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

image

image

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/service-portal/core/src/components/VideoPlayer/VideoPlayer.tsx [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+                    icon={isMuted ? 'volumeMute' : 'volumeHigh'}
+                    color="blue400"
+                  />
+                </button>
+              </Box>
+            </GridColumn>
+            <GridColumn span="1/12" className={styles.audioControl}>
+              <Box display="flex" justifyContent="center">
+                <button
+                  title={isFullscreen ? 'Close fullscreen' : 'Open fullscreen'}
```
Comment (thordurhhh) [inline libs/service-portal/core/src/components/VideoPlayer/VideoPlayer.tsx:200]: Add aria-label?

File: libs/service-portal/core/src/components/VideoPlayer/VideoPlayer.tsx [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+                progress={progress}
+                onClick={setTimeChange}
+                renderProgressBar={isReady}
+                variant
+              />
+            </GridColumn>
+            <GridColumn span="1/12" className={styles.audioControl}>
+              <Box display="flex" justifyContent="center">
+                <button
+                  title={isMuted ? 'Unmute video' : 'Mute video'}
```
Comment (thordurhhh) [inline libs/service-portal/core/src/components/VideoPlayer/VideoPlayer.tsx:187]: Add aria-label?

File: libs/service-portal/core/src/components/VideoPlayer/VideoPlayer.tsx [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+                display="flex"
+                justifyContent="spaceAround"
+                alignItems="center"
+                columnGap={'p2'}
+              >
+                {isReady && (
+                  <button
+                    disabled={!isReady}
+                    title={
+                      hasEnded
```
Comment (thordurhhh) [inline libs/service-portal/core/src/components/VideoPlayer/VideoPlayer.tsx:147]: Add aria-label?

File: libs/service-portal/core/src/components/VideoPlayer/VideoPlayer.css.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  maxWidth: '100%',
+  aspectRatio: '1.55/1',
+  borderRadius: theme.border.radius.large,
+  borderWidth: theme.border.width.large,
+  borderStyle: theme.border.style.solid,
+  borderColor: theme.border.color.blue200,
+
+  ...themeUtils.responsiveStyle({
+    sm: {
+      width: '546px',
```
Comment (thordurhhh) [inline libs/service-portal/core/src/components/VideoPlayer/VideoPlayer.css.ts:14]: What "random" number is this? 😵‍💫

File: libs/service-portal/core/src/components/AudioPlayer/AudioPlayer.tsx [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+                  />
+                </button>
+              </Box>
+            </GridColumn>
+            {/*}
+    <GridColumn span="1/12">
+      <Box display="flex" justifyContent="center">
+        <Icon icon="ellipsisVertical" color="blue400" />
+      </Box>
+    </GridColumn>
```
Comment (thordurhhh) [inline libs/service-portal/core/src/components/AudioPlayer/AudioPlayer.tsx:141]: Remove?

File: libs/service-portal/core/src/components/AudioPlayer/AudioPlayer.tsx [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+                progress={audioProgress}
+                onClick={setTimeChange}
+                renderProgressBar={isReady}
+                variant
+              />
+            </GridColumn>
+            <GridColumn span="1/12">
+              <Box display="flex" justifyContent="center">
+                <button
+                  title={isMuted ? 'Unmute audio' : 'Mute audio'}
```
Comment (thordurhhh) [inline libs/service-portal/core/src/components/AudioPlayer/AudioPlayer.tsx:126]: Add aria-label?

File: libs/service-portal/core/src/components/AudioPlayer/AudioPlayer.tsx [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+      />
+      <Box display={isReady ? 'block' : 'none'}>
+        <GridContainer className={styles.container}>
+          <GridRow className={styles.audio} align="center" alignItems="center">
+            <GridColumn span="1/12">
+              <Box display={'flex'} justifyContent="center">
+                {
+                  <button
+                    title={
+                      hasEnded
```
Comment (thordurhhh) [inline libs/service-portal/core/src/components/AudioPlayer/AudioPlayer.tsx:85]: add aria-label?

**Review by thordurhhh — COMMENTED** [review]
Looking good. Minor comments.

File: libs/service-portal/core/src/components/AudioPlayer/AudioPlayer.css.ts [hunk 8]
```diff
@@ -0,0 +1,21 @@
+import { theme, themeUtils } from '@island.is/island-ui/theme'
+import { style } from '@vanilla-extract/css'
+
+export const container = style({
+  maxWidth: '100%',
+  marginLeft: 0,
+
+  ...themeUtils.responsiveStyle({
+    sm: {
+      width: '546px',
```
Comment (thordurhhh) [inline libs/service-portal/core/src/components/AudioPlayer/AudioPlayer.css.ts:10]: 👀

**Review by thordurhhh — APPROVED** [review]
Nice. Title and aria labels should probably be translated.
One `'546px'` in the audioplayer 👀

File: libs/service-portal/core/src/components/ProgressBar/ProgressBar.css.ts [hunk 9]
```diff
@@ -0,0 +1,14 @@
+import { style } from '@vanilla-extract/css'
+
+export const inner = style({
+  bottom: 0,
+  left: 0,
+  right: 0,
+  top: 0,
+  transition: 'transform 0.5s ease-out',
```
Comment (Tryggvig) [inline libs/service-portal/core/src/components/ProgressBar/ProgressBar.css.ts:8]:
```suggestion
  transition: 'transform 0.2s linear',
```

File: libs/service-portal/core/src/components/AudioPlayer/AudioPlayer.css.ts (hunk 8 above)
Comment (Tryggvig) [inline libs/service-portal/core/src/components/AudioPlayer/AudioPlayer.css.ts:10]:
```suggestion
      width: theme.breakpoints.sm,
```

**Review by Tryggvig — CHANGES_REQUESTED** [review]

**Review by Tryggvig — APPROVED** [review]

**Pull request merged**